src/manual_control.cpp

```diff
@@ -38,7 +41,22 @@ void joystickCallback(const isc_joy::xinput::ConstPtr& joy){
 	msg.angular.z = enableDriving ? joyTurn : 0;
 	manualPub.publish(msg);
 
-	if(enableLogging) ROS_INFO("Manual Control: %s linear.x=%f angular.z=%f", joy->LB ? "on" : "off", msg.linear.x, msg.angular.z);
+	// send drive mode signal to robot state controller
+	std_msgs::Bool driveModeSignal;
+	if (joy->Start) { //The Start button has been pressed
+		startButtonDown = true;
+		driveModeSignal->data = true;
+		driveModeSignalPub.publish(driveModeSignal);
+	} else if (startButtonDown && !joy->Start) {
```

Comment:
This will publish `true` while Start is pressed and publish `false` when it is released. Is this intentional? The [previous implementation](https://github.com/iscumd/Yeti/blob/master/yeti_snowplow/src/yeti_drive_mode_control.cpp#L53) is a toggle upon release of the button.

Also note that if I were to hold down Start and move the sticks/press other buttons, it will publish `true` for each of those inputs while Start is held.

Reply:
Good catch, not sure what I was thinking.